step/step.go

```diff
@@ -271,10 +274,22 @@ func (c combined) run(prefix string) bool {
 	return true
 }
 
+func Value(desc, s string) Step {
```

Comment:
Can we have a comment here what Value does/when to use it? sounds like it's for returning the outcome of a step?

Reply:
Sure